### PR TITLE
Music: default blocks mode to simple2

### PR DESCRIPTION
**Pull request by breville**
Default the blocks mode to `simple2`. 

Also, hide instructions and instructions icon until instructions are successfully loaded.  And, for now, only attempt to load instructions if `?show-instructions=true` is specified.

Also adjusts the toolbox categories:

Screenshot 2023-02-17 at 9 08 01 AM

Screenshot 2023-02-17 at 9 08 07 AM

Screenshot 2023-02-17 at 9 08 33 AM

### CODE REVIEW

**Review by mikeharv — APPROVED** [review]

File: apps/src/music/appConfig.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {BlockMode} from './constants';
  * a default if an invalid value or no value is found.
  */
 export const getBlockMode = () => {
-  const defaultMode = BlockMode.TRACKS; // Current default is Tracks
+  const defaultMode = BlockMode.SIMPLE2; // Current default mode
```
Comment (sanchitmalhotra126) [inline apps/src/music/appConfig.js]: "Current default is Simple2"? 🙂

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/music/appConfig.js (hunk 1 above)
Comment (breville) [reply apps/src/music/appConfig.js]: My thinking was that the code was the source of truth and then we wouldn't need to change the comment each time we changed the value.

Comment (sanchitmalhotra126) [reply apps/src/music/appConfig.js]: Haha yeah that's true, we can probably just delete this comment then

File: apps/src/music/views/MusicView.jsx [hunk 2]
```diff
@@ -109,12 +109,15 @@ class UnconnectedMusicView extends React.Component {
       Globals.setPlayer(this.player);
     });
 
-    this.loadInstructions().then(instructions => {
-      this.setState({
-        instructions: instructions,
-        showInstructions: !!instructions
+    // Disable instructions for now.
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/MusicView.jsx]: Hm should this be behind a param in `appConfig` so we can still turn it back on if needed?

Comment (breville) [reply apps/src/music/views/MusicView.jsx]: Good call; will do.

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**